index.html

```diff
@@ -454,10 +454,11 @@ <h2 class="contact-title">Contact me</h2>
                 />
                 <label for="email" class="form-label" hidden>Email</label>
                 <input
+
                   type="email"
                   id="email"
                   placeholder="Enter you email"
-                  class="form-control mb-4"
+                  class="email form-control mb-4"
```

Comment:
- [ ] Please I highly recommend adding a `required` attribute to the `textarea` for validation. 😃